### PR TITLE
modify ssms release note to include a bug fix

### PR DESCRIPTION
**Pull request by MSraye**
modify ssms release note to include a bug fix in 18.12

### CODE REVIEW

**Comment by PRMerger15** [comment]
@MSraye : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by erinstellato-ms — APPROVED** [review]

**Comment by erinstellato-ms** [comment]
#sign-off

File: docs/ssms/release-notes-ssms.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ Download previous SSMS versions by selecting the download link in the related se
 |----------|---------|
 | Availability Group Dashboard | Fixed issue when connecting to the Availability Group Dashboard for an AG on SQL Server 2016 which resulted in "unknown property ClusterType" error.  See [SSMS 18.11.1 Bug](https://feedback.azure.com/d365community/idea/1603d3a2-73ab-ec11-a81c-6045bd7d1bee). |
 | Analysis Services | Corrected behavior of SSMS not saving Analysis Services settings separately. |
+| Integration Services | Fixed an issue which caused the error, "System.ArgumentException: Value does not fall within the expected range" that would be encountered unexpectedly at step "Changing protection level" when deploying SSIS project. |
```
Comment (Court72) [inline docs/ssms/release-notes-ssms.md:120]:
```suggestion
| Integration Services | Fixed an issue that caused the error "System.ArgumentException: Value does not fall within the expected range" that would be encountered unexpectedly at step "Changing protection level" when deploying SSIS project. |
```

**Comment by PRMerger13** [comment]
@Court72 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**